docs/test-plans/001-Big-Blocks/test-cases/tc-003-full-sync-past.md

```diff
@@ -48,14 +48,14 @@
 2. Receives the trusted genesis hash and ip from Bridge Nodes
 3. Waits until `N` amount of block has been produced by the chain
 4. Starts syncing the chain afterwards
-5. Checks that it can catch up the chain faster then new blocks are produced (\*)
+5. Checks that it can catch up the chain faster than new blocks are produced (\*)
```

Comment:
tagging #79 and #89 for reference, as they would enable the required measurement
> We need to measure the sync time to have a baseline for further benchmarking of the new p2p stack that is implemented